service/league_service/league_rater.py

```diff
@@ -34,10 +34,11 @@ def rate(
             return cls._do_placement(league, current_score, rating)
 
         player_div = league.get_division(current_score.division_id)
-        if player_div is None:
+        if player_div is None or current_score.score is None:
             cls._logger.warning(
-                "Doing placement again, because a division for id %s could not be found.",
-                current_score.division_id,
+                "Doing placement again, because a division not found or score set to null. "
+                "Current league score: %s",
+                current_score,
```

Comment:
needs to be current_score.score, right?

Reply:
I wanted it to Print something like `LeagueScore(1, None, 10)` or similar.

Reply:
Looks like this:
```
------------------------------------------------------------------------------------------------ Captured log call -------------------------------------------------------------------------------------------------
WARNING  LeagueRater:league_rater.py:41 Doing placement again, because a division not found or score set to null. Current league score: LeagueScore(division_id=2, score=None, game_count=10)
```